locales/result/br.json

```diff
@@ -17,7 +17,7 @@
   "mail.payment.support": "User request support for Task",
   "mail.assign.owner.subject": "Alguém demonstrou interesse por sua tarefa no Gitpay",
   "mail.assign.owner.hello": "Olá",
-  "mail.assign.owner.main": "{{name}} is interested to work on the task <a href=\"{{url}}\">{{title}}</a>. You can contact him on {{email}} to know more.",
+  "mail.assign.owner.main": "{{name}} is interested to work on the task <a href=\"{{url}}\">{{title}}</a>.",
```

Comment:
You shouldn't change the result files, they're managed by the translation library.